codex/uberdev-codex/lib/run_manifest.py

```
@@ -30,7 +32,7 @@
 import time
 from contextlib import contextmanager
 from dataclasses import dataclass
-from typing import Any, Iterable
+from typing import Any, Callable, Iterable
```

Ruff (0.15.21)

[warning] Import from `collections.abc` instead: `Callable`, `Iterable`

Import from `collections.abc`

(UP035)